controller/space.go

```diff
@@ -201,7 +201,8 @@ func (c *SpaceController) Delete(ctx *app.DeleteSpaceContext) error {
 			"space_id": spaceID,
 		}, "no configuation found in SpaceController object")
 		return jsonapi.JSONErrorResponse(
-			ctx, errors.NewInternalError(ctx, spaceDeletionErrorExternal))
+			ctx, errors.NewInternalErrorFromString("could not delete space"),
```

Comment:
the message seems off

Reply:
This is the error the client would receive. `Missing configuration for space controller` error wouldn't make sense to the client. That's why I have used a generic error like `could not delete space`